src/lib/viewers/controls/color-picker/__tests__/ColorPickerPalette-test.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
 import ColorPickerPalette from '../ColorPickerPalette';
+import { AnnotationColor } from '../../../../AnnotationModule';
```

Comment:
We should avoid importing annotations-specific code into a generic component or its tests, if possible.